### ISSUE TITLE
Integrated Dynamics display Slows down\Freezes game with lists longer than 20

### ISSUE THREAD

**Issue by Ajikozau** [FTBTeam/FTB-Modpack-Issues#5804]
### Modpack

FTB NeoTech

### Modpack version

1.7.0

### Launcher

FTB App

### Has the pack been modified

No

### Describe the bug

Whenever you insert a card in a ID Display to display a list longer than 20 (any kind of list), the game slows down, if you try to do this with 2 displays the game freezes and has to be manually closed.

### Steps to reproduce

Craft a display. Craft a card with a list of 20 (any). Put it into the display.
Craft a second display. Craft a second card (or copy the first). Put it into the second display.

### Expected behaviour

Game shouldn't freeze or slow down.

**Comment by Ajikozau** [comment]
The issue is caused by some kind of incompatibility between LogBegone and Integrated Dynamics. Deactivating LogBegone solves the issue.

**Comment by ArtpokeBr** [comment]
Yup, seems to still be a thing, so Logs be Gone is now Removed from the Pack.
